metrics.go

```diff
@@ -50,6 +51,25 @@ func flattenMetrics(data metrics, memo flatMetrics, prefix string) flatMetrics {
 	return memo
 }
 
+func decodeSyncData(data metrics, prefix string) flatMetrics {
+	var memo = make(flatMetrics)
+	for k, v := range data {
+		key := prefix + normalizeKey(k) + "_value"
+		value := decodeHexAddr(v.(string))
+		memo[key] = value
+	}
+	return memo
+}
+
+func decodeHexAddr(s string) string {
+	/* Prometheus needs a number, not a hex */
+	number, err := strconv.ParseUint(s, 0, 64)
```

Comment:
Why `uint` and not `int`?

Reply:
Because block number cant be in the negatives? I can change it, I just don't see why.